src/functions-and-arrays.js

```diff
@@ -1,41 +1,140 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(firstNumber, secondNumber) {
+  if (firstNumber > secondNumber) {
+    return firstNumber;
+  } else if (secondNumber > firstNumber) {
+    return secondNumber;
+  } else {
+    return firstNumber;
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(arr) {
+  let largest = 0;
+  let longest = '';
+
+  if (arr.length > 0) {
+    for (i = 0; i < arr.length; i++) {
+      if (arr[i].length > largest) {
+        largest = arr[i].length;
+        longest = arr[i];
+      } else {
+        continue;
+      }
+    }
+    return longest;
+  } else {
+    return null;
+  }
+}
```

Comment:
Boa! Uma dica q eu daria é, quando for uma verificação de "validação dos dados de entrada", deixar ela antes do seu código. Nesse caso é um código pequeno então não teria problema, mas com um código maior ou com mais verificações isso pode ficar meio confuso.
exemplo:
```javascript
function findLogestWord(arr) {
  if(arr.length === 0){
    return null;
  }
  // restante do código.
}
```